pkg/gui/refresh.go

```diff
@@ -553,7 +570,12 @@ func (gui *Gui) refreshStatus() {
 	}
 
 	name := presentation.GetBranchTextStyle(currentBranch.Name).Sprint(currentBranch.Name)
-	repoName := utils.GetCurrentRepoName()
+	var repoName string
+	repoName = utils.GetCurrentRepoName()
```

Comment:
```suggestion
	repoName := utils.GetCurrentRepoName()
```
more concise than declaring the variable on a separate line

Reply:
Thanks, I know better :P